src/screens/OtherProfile/index.js

```diff
@@ -164,13 +158,6 @@ const OtherProfile = () => {
   }, []);
 
   React.useEffect(() => {
-    netInfo.addEventListener((state) => {
-      if (state.isConnected) {
-        initData();
-      } else {
-        handleOfflineMode();
-      }
-    });
     initData();
   }, []);
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the network connectivity check might lead to issues if the device is offline. If the app should work offline, then this change is fine. Otherwise, consider adding back the network connectivity check.



<!-- This is an auto-generated comment by OSS CodeRabbit -->